packages/block-library/src/gallery/style.scss

```diff
@@ -3,6 +3,10 @@
 
 // Styles for current version of gallery block.
 .wp-block-gallery.has-nested-images {
+	--gallery-block--gutter-size: var(
+		--wp--style--block-gap,
+		$blocks-block__margin
```

Comment:
> Should this fallback value be #{$grid-unit-20} for consistency with the other fallback values below? I think $blocks-block__margin is a Sass variable used only in the Button/Buttons blocks?

🤦 Doh, thanks, was a bit confused there - have tidied it up so that the correct fallback `$grid-unit-20` is used and it is just declared in one place

Reply:
Thanks @glendaviesnz! I think this one also needs the [interpolation syntax](https://sass-lang.com/documentation/breaking-changes/css-vars) to render the fallback Sass value. In the CSS the Sass variable is being rendered as-is, instead of being swapped out for the real value:

![image](https://user-images.githubusercontent.com/14988353/132442332-d8278308-20f0-486b-8e65-f734a90a93b9.png)

So it'd be `#{$grid-unit-20}` instead of `$grid-unit-20`, which should result in the following:

![image](https://user-images.githubusercontent.com/14988353/132442551-83b4bae7-6c01-4c3b-a153-711ef79a18a6.png)